openlibrary/plugins/openlibrary/js/page-barcodescanner/index.js

```diff
@@ -48,11 +49,18 @@ export function init() {
     });
 
     let lastResult = null;
+    let confirmation_list = [];
     Quagga.onDetected(result => {
         const code = result.codeResult.code;
+        const url = `/isbn/${code}`;
+
         if (!isBarcodeISBN(code) || code == lastResult) return;
         lastResult = code;
```

Comment:
```suggestion
    let confirmation_list = [];
    Quagga.onDetected(result => {
        const code = result.codeResult.code;
        const url = `/isbn/${code}`;

        if (!isBarcodeISBN(code)) return;
```

Reply:
I think that `lastResult` can be removed since we are checking for redundant codes with the `checkISBN` function.  Also, it looks like multiple `LazyBookCard`s  could be created with the same ISBN using this approach with multiple scans enabled.
e.g.
Code 1234567890 is scanned.
Then, code 2345678901 is scanned.
Finally, code 1234567890 is scanned again.